client.go

```diff
@@ -53,8 +53,16 @@ type Client interface {
 	// DestroyVolumes returns an error if any of the volume deletion fails. It does not
 	// return an error if volumes were not found on the server.
 	// DestroyVolumes returns an error as to why one or more volumes could not be deleted.
-	// DestroyVolumes return list of volumes that are destroyed successfully.
 	DestroyVolumes(lager.Logger, []string) error
+
+	// DestroyVolume deletes the volume with the provided handle that is present on the server.
+	//
+	// You are required to pass in a logger to the call to retain context across
+	// the library boundary.
+	//
+	// DestroyVolume returns an error if the volume deletion fails. It does not
+	// return an error if the volume was not found on the server.
+	DestroyVolume(lager.Logger, string) error
```

Comment:
Probably cleaner to have the singular destroy method available anyway, but couldn't the caller just call `DestroyVolumes` with a single item? Or is there some difference in how they're handled that we'd like to avoid?

Reply:
@vito It just seemed cleaner. There already was a `DestroyVolume` method in `volume_server.go`, all we had to do was add a wrapper in `client/client.go` so it was fairly straightforward.